tests/interfaces/records/handlers/records-query.spec.ts

```diff
@@ -84,6 +85,37 @@ describe('handleRecordsQuery()', () => {
       expect(reply2.entries?.length).to.equal(1); // only 1 entry should match the query
     });
 
+    it('should be able to query by attester', async () => {
+      // scenario: 3 records authored by alice, 1st attested by alice, 2nd attested by bob, 3rd attested by carol
+      const alice = await DidKeyResolver.generate();
+      const bob = await DidKeyResolver.generate();
+      const carol = await DidKeyResolver.generate();
+      const recordsWrite1 = await TestDataGenerator.generateRecordsWrite({ requester: alice, attesters: [alice] });
+      const recordsWrite2 = await TestDataGenerator.generateRecordsWrite({ requester: alice, attesters: [bob] });
+      const recordsWrite3 = await TestDataGenerator.generateRecordsWrite({ requester: alice, attesters: [carol] });
```

Comment:
Can this be removed? Seems like there are no expectations for query-ing with `carol` as the attester.

Reply:
Sure, removed, adapted the test to include the variation recommended beblow.